bin/load-spark-env.sh

```diff
@@ -20,7 +20,29 @@
 # This script loads spark-env.sh if it exists, and ensures it is only loaded once.
 # spark-env.sh is loaded from SPARK_CONF_DIR if set, or within the current directory's
 # conf/ subdirectory.
-FWDIR="$(cd "`dirname "$0"`"/..; pwd)"
+realpath () {
```

Comment:
Hi @vanzin , the behavior of `readlink -f` is different under Mac, here is the discussion in [stackoverflow](http://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac). Another solution is to install `greadlink` through brew.